_IBC/paxi-cosmoshub.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "paxi",
```

Comment:
p comes after c, so paxi must be chain2 and must come last in the file name. (it how we avoid multiple files describing the same pair). Same issue with the osmosis Ibc file